src/status_im/translations/fr.cljs

```diff
@@ -148,167 +141,225 @@
+
+   ;; wallet
+   :wallet                                "Portefeuille"
+   :wallets                               "Portefeuilles"
+   :your-wallets                          "Vos portefeuilles"
+   :main-wallet                           "Portefeuille principal"
+   :wallet-error                          "Erreur survenue lors du chargement de données"
+   :wallet-send                           "Envoyé"
+   :wallet-request                        "Requête"
+   :wallet-exchange                       "Echange"
+   :wallet-assets                         "Actifs"
+   :wallet-add-asset                      "Ajouter un actif"
+   :wallet-total-value                    "Valeur totale"
+   :wallet-settings                       "Paramètres du portefeuille"
+   :signing-phrase-description            "Signez la transaction en saisissant votre mot de passe. Assurez-vous que les mots ci-dessus correspondent à votre phrase de signature secrète"
+   :wallet-insufficient-funds             "Fonds insuffisants"
+   :request-transaction                   "Effectuer une transaction"
+   :send-request                          "Envoyer la requête"
+   :share                                 "Partager"
+   :eth                                   "ETH"
+   :currency                              "Devise"
+   :usd-currency                          "USD"
+   :transactions                          "Transactions"
+   :transaction-details                   "Détails de la transaction"
+   :transaction-failed                    "transaction échouée"
+   :transactions-sign                     "Signer"
+   :transactions-sign-all                 "Tout signer"
+   :transactions-sign-transaction         "Signer la transaction"
+   :transactions-sign-later               "Signer ultérieurement"
+   :transactions-delete                   "Supprimer la transaction"
+   :transactions-delete-content           "La transaction va être enlevée de la list 'Non-Signées'"
+   :transactions-history                  "Historique"
+   :transactions-unsigned                 "Non-Signées"
+   :transactions-history-empty            "Aucune transaction dans votre historique pour le moment"
+   :transactions-unsigned-empty           "Vous n'avez aucune transaction non signée"
+   :transactions-filter-title             "Filtrer l'historique"
+   :transactions-filter-tokens            "Jetons"
```

Comment:
Translating Tokens in Jetons does sound strange to me.

Reply:
I can leave it as `Tokens` if it makes more sense.